README.rst

```diff
@@ -20,6 +20,13 @@ If you want to test out a change before you submit it, download and install
 
     $ pip install lektor
 
+If you use `pipenv`_, a .Pipfile is provided that has Lektor and Python 2.7
```

Comment:
Hi @afs2015 now that we are adding this, maybe it is a good idea to include a couple of subsections

Installing development environment
------------------------------------

Using pip
~~~~~~~


Using pipenv
~~~~~~~~~